CONTRIBUTING.rst

```diff
@@ -146,11 +146,50 @@ Deploying
 
 A reminder for the maintainers on how to deploy.
 
-1.  Commit all your changes.
-2.  Run ``tbump X.Y.Z``.
-3.  Wait a bit that the release on github and zenodo is created.
-4.  Run ``make README.rst`` to update the citation information in the README from the
-    zenodo API. Check if the version is actually correct, otherwise grab a tea and
-    wait a little more for zenodo to mint the new version. Once it worked, commit the
-    change.
-5.  Upload the release to pyPI: ``make release``
+
+1 Run ``tbump``
+~~~~~~~~~~~~~~~~
+
+-  commit all your changes
+-  make sure `gh`_ is installed on your system
+-  Decide what the new version number should be
+-  For version X.Y.Z - increase X for a major release, increase Y when breaking changes are introduced, increase Z for minor changes
+-  Run ``venv/bin/tbump X.Y.Z`` if you don’t have tbump installed globally
```

Comment:
```suggestion
-  Run ``venv/bin/tbump X.Y.Z``
```
Using the `tbump` in the venv is always correct, otherwise it might be an old version which isn't compatible with our config.